templates/ac_local.json.tmp

```diff
@@ -12,8 +12,8 @@
         "admin_token": "<<ac_admin_token>>",
         "token_secret": "<<ac_token_secret>>",
         "ssl": {
-            "verify_certs": "<<ac_ssl_verify_certs>>",
-            "hostname_validation": "<<ac_ssl_hostname_validation>>",
+            "verify_certs": <<ac_ssl_verify_certs>>,
```

Comment:
why removing double quote?

Reply:
I tested many times and with double quotes, it will generate errors.  I checked the default.json and there is no double quotes for these two values.

Reply:
"false" is not always the same as false. Most of developers will think they the same. false is boolean. "false" is a string. If a program detects the field with proper conversion build-in, "false" can be the same as false. Otherwise,  "false" is true in boolean. Beside my own testing, I noticed Mike ask me to remove the double quote in few case before we changed ssl values.

Reply:
ok.